pkgs/tools/security/notary/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchgit, buildGoPackage, git, libtool }:
+
+buildGoPackage rec {
+  name = "notary-${version}";
+  version = "0.4.3";
+  gitcommit = "9211198";
+
+  src = fetchgit {
+    url = "https://github.com/theupdateframework/notary";
+    rev = "refs/tags/v${version}";
+    sha256 = "1max6rgangmw892l8zpijhx5bj9zllqim457k1dq7vf1mzybihr6";
+    leaveDotGit = true;
+  };
+
+  buildInputs = [ git libtool ];
+
+  goPackagePath = "github.com/docker/notary";
```

Comment:
The move to `theupdateframework` (due to giving the project to CNCF) happened after the `v0.4.3` release.. so building imports, etc.. are still refering the path with `docker`.